stable/cert-manager/templates/00-namespace.yaml

```diff
@@ -0,0 +1,6 @@
+{{ if .Values.createNamespaceResource }}
```

Comment:
What's the point of this? Helm creates namespaces automatically, so this has to fail.

Reply:
It is used when generating the 'static manifests' for those that don't want to use helm (https://github.com/jetstack/cert-manager/tree/master/contrib/manifests/cert-manager).

We use `helm template` to generate these files, hence adding it as an option in the Helm chart makes it easy for us to include a namespace resource.

Reply:
Still it doesn't make sense in the Helm chart. Setting it to true will cause an error.